src/style.css

```
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  --moyo-blue: #00acdc;
+}
+
+.picture-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.picture-container img {
```

```suggestion
.picture-container .product-card__illustration  {
```
your image has class, don't style tags

---

I made it to use it of any picture container. To picture everytime will sized by father, not bigger and not stretched. And to be centered.